docs/index.md

```diff
@@ -1204,6 +1204,13 @@ the statement(s) to move using the left-hand-side check box, then
 move them up and down using the "up" and "down" icons.
 However, there's no need to reorder these statements.
 
+**Important**: Every hypothesis (and goal) id is a database label, so it
+*must* be unique in the database.
+It cannot match any math symbol token (like `1`), assertion label,
+or a label of any other hypotheses.
+The convention in `set.mm`, as shown above, is for hypotheses to be labelled
+as the name of the goal + "." + an integer starting from 1.
+
```

Comment:
`Every hypothesis (and goal) id is a database label, so it *must* be unique in the database.`

I also would clarify the following:

```
But you don't have to worry about incorrect labels. 
Metamath-lamp validates labels you use 
and it will show an error message if one of the labels is incorrect.
```